src/views/Calculator.vue

```diff
@@ -0,0 +1,487 @@
+      this.selectedDownpayment = this.downpaymentCalculations.filter(
+        (result) => {
+          return result.bizId == bizId && result.percent == percent;
+        }
+      )[0];
+
+    },
+
+    downpaymentCalc() {
+      let downPaymentArr = [];
+      this.businessTypes.forEach((bizType) => {
+        return this.downPaymentRates.forEach((paymentRate) => {
+          let filteredBizType = this.calculation.filter((param) => {
+            return (
+              bizType.id === param.business_type_id &&
+              paymentRate.id === param.down_payment_rate_id 
+            );
+          });
+
+          const { total, actualDownpayment, actualRepayment } = calculate(
+            this.selectedProduct.price,
+            paymentRate,
+            filteredBizType[1]
+          );
+
+          downPaymentArr.push({
+            bizId: bizType.id,
+            percent: paymentRate["percent"],
+            total,
+            actualDownpayment,
+            actualRepayment,
+          });
+        });
+      });
+      this.downpaymentCalculations = downPaymentArr;
+      return downPaymentArr;
+    },
+
+    async getProduct() {
+      try {
+        const fetchProduct = await get(this.apiUrls.getProduct + this.product);
+        this.products = fetchProduct?.data?.data?.data;
+        this.products.find((item) => {
+          return item.product_id == this.selectedProduct.product_id;
+        });
+      } catch (err) {
+        this.$displayErrorMessage(err);
+      }
+    },
+    async getDownPaymentRates() {
+      try {
+        const fetchDownPaymentRates = await get(this.apiUrls.downPaymentRates);
+        this.downPaymentRates = fetchDownPaymentRates?.data?.data?.data;
+        this.downPaymentRates = this.downPaymentRates.filter((item) => {
+          return !(
+            item.name.includes("plus") ||
+            item.name.includes("zero") ||
+            item.name.includes("ten")
+          );
+        });
+        this.downPaymentRates = this.downPaymentRates.sort((a, b) => {
+          return a.percent - b.percent;
+        });
+      } catch (err) {
+        this.$displayErrorMessage(err);
+      }
+    },
+
+    async getCalculation() {
+      try {
+        const fetchGetCalclations = await get(this.apiUrls.getCalculation);
+        const unwrapped = fetchGetCalclations?.data?.data;
+        this.calculation = unwrapped;
+      } catch (err) {
+        this.$displayErrorMessage(err);
+      }
+    },
+    async getBusinessTypes() {
+      try {
+        const fetchBusinessTypes = await get(this.apiUrls.businessTypes);
+        this.businessTypes = fetchBusinessTypes?.data?.data?.data;
+        this.businessTypes = this.businessTypes.filter((item) => {
+          return item.name.includes("Products");
+        });
+      } catch (err) {
+        this.$displayErrorMessage(err);
+      }
+    },
+  },
+};
+</script>
+<style scoped>
+@import url("https://fonts.googleapis.com/css2?family=Nanum+Pen+Script&display=swap");
```

Comment:
why are we importing this google font here?

Reply:
it is used on the UI for mobile